content/docker-hub/company-faqs.md

```diff
@@ -22,36 +22,40 @@ No, you can continue with business as usual.
 
 ### How many company owners can I add?
 
-A maximum of 10 company owners can be added to a single company account.
+You can add a maximum of 10 company owners to a single company account.
 
-### What permission does the company owner have in the associated/nested organizations?
+### What permissions does the company owner have in the associated/nested organizations?
 
-Company owners can navigate to the **Organizations** page to view all their nested organizations in a single location. They can also view or edit organization members and change SSO and SCIM settings. Changes to company settings impact all users in each organization under the company.
+Company owners can navigate to the **Organizations** page to view all their nested organizations in a single location. They can also view or edit organization members and change SSO and SCIM settings. Changes to company settings impact all users in each organization under the company. See [Roles and permissions](../security/for-admins/roles-and-permissions.md).
 
 ### What features are supported at the company level?
 
-Domain verification, Single Sign-on, and System for Cross-domain Identity Management (SCIM) are supported at the company level. The following aren't supported:
+You can manage domain verification, Single Sign-on, and System for Cross-domain Identity Management (SCIM) at the company level. You can manage the following features at the organization level:
 
 - Image Access Management
 - Registry Access Management
 - User management
 - Billing
 
+You can manage users at the company level when you use Docker Admin. 
```

Comment:
@stephaurelio does this refer to User Management view? Would we want to be explicit here, i.e: "For helping you to view and manage users across all the organization under your company we're introducing User Management -> link to user management docs"

Reply:
Yes, will update to link to the manage users with a company in Docker Admin docs. Thanks!